.github/workflows/ruby.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will download a prebuilt Ruby version, install dependencies and
+# run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby: [2.5, 2.6, 2.7, "3.0"]
```

Comment:
I have one question: why is only `3.0` surrounded with `"`?

Reply:
This is to make the Ruby version appear as 3.0 in the build results. GitHub Actions displays the other numbers correctly automatically.

Compare the left-hand side of this display: https://github.com/mvz/gir_ffi/actions/runs/476007991

With this one: https://github.com/mvz/happymapper/runs/1667668414

I suppose you could surround all of them with quotes to be consistent.